
### Enable TX Queue support for better performance
### Enable PTP (Precision Time Protocol) support (if enabled in kernel)
### Enable RSS option for better performance
### Enable automatic firmware loading, if optional firmware is available
### Enable WOL magic packet by default

--- a/src/Makefile	2022-10-02 02:41:33.318007620 +0000
+++ b/src/Makefile	2022-10-02 02:42:18.302129039 +0000
@@ -39,14 +39,14 @@
 ENABLE_S5WOL = y
 ENABLE_S5_KEEP_CURR_MAC = n
 ENABLE_EEE = y
-ENABLE_S0_MAGIC_PACKET = n
+ENABLE_S0_MAGIC_PACKET = y
 ENABLE_TX_NO_CLOSE = y
-ENABLE_MULTIPLE_TX_QUEUE = n
-ENABLE_PTP_SUPPORT = n
-ENABLE_PTP_MASTER_MODE = n
-ENABLE_RSS_SUPPORT = n
+ENABLE_MULTIPLE_TX_QUEUE = y
+ENABLE_PTP_SUPPORT = y
+ENABLE_PTP_MASTER_MODE = y
+ENABLE_RSS_SUPPORT = y
 ENABLE_LIB_SUPPORT = n
-ENABLE_USE_FIRMWARE_FILE = n
+ENABLE_USE_FIRMWARE_FILE = y
 DISABLE_PM_SUPPORT = n
 DISABLE_MULTI_MSIX_VECTOR = n
 
